src/game.cpp

```diff
@@ -11584,29 +11583,33 @@ void game::chat()
         }
     }
 
-    if (available.empty()) {
-        add_msg(m_info, _("There's no-one close enough to talk to."));
-        if ( !query_yn(_("No one nearby. Yell?")) ) {
-            return;
-        }
-        u.shout();
-
-    } else if (available.size() == 1) {
+    if( available.size() == 1 ) {
```

Comment:
As for the `available.size() == 1` - it was like this before and currently there is "Y - yell" in NPC menu. The only mechanics change here is the `available.empty()` case - currently (in master) it has a separate y/n prompt, here it allows yell/cancel (for muscle memory and consistency).

I could move it all to this menu and remove the "exactly one" case.